vignettes/social_gain_vignette/readme.md

```diff
@@ -109,35 +108,35 @@ A successful precision feedback intervention leveraging the social gain pathway
 ## Message templates
 Message templates represent possible motivational messages that a precision feedback system can send. There are three message templates that the precision feedback system can access for this vignette: 
 
-**[Top 10% Peer Benchmark](https://i.imgur.com/I1EeK7L.jpeg)** <sub>Todo - JSON template in Github, link here</sub> message:
+The `Achieved top 10% peer benchmark` [<sub>(GH)</sub>](https://github.com/Display-Lab/knowledge-base/blob/main/message_templates/achieved_top_10_peer_benchmark.json) message reads:
```

Comment:
Making sure message templates have consistent names across all platforms, including the vignettes